### ISSUE TITLE
Install Error in Blender 3.2.0

### ISSUE THREAD

**Issue by SJT1988** [InamuraJIN/ActionRecorder#31]
```
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 3.2\3.2\scripts\modules\addon_utils.py", line 335, in enable
    mod = __import__(module_name)
ModuleNotFoundError: No module named 'ActionRecorder-1'
```

**Comment by InamuraJIN** [comment]
New version released

**Comment by RivinHD** [comment]
Fixed with [Release 4.0.2](https://github.com/InamuraJIN/ActionRecorder/releases/tag/4.0.2)
